Gemfile

```diff
@@ -7,9 +7,8 @@ gem 'pry-nav'
 gem 'pry-rescue'
 
 # optional dependency for more accurate timezone conversion
-gem 'tzinfo', '1.2.5'
+gem 'tzinfo', '2.0.6'
 # gem 'tzinfo', '2.0.0'
 
 # required for CircleCI to build properly with ruby 1.9.3
```

Comment:
Can we yeet while here?

Reply:
I think this comment applied to bot the json dependency and the rack dependency. Both are code that exists in the Ruby core. So it might make sense to depend on the standard core in both cases. In any case, we certainly don't use CircleCI for CI anymore.